### PR TITLE
Add mini map to search filter

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Locations :)

### Changes proposed in this pull request
Add small static image of map to results view

### Guidance to review
screen shot 2018-11-21 at 12 44 56

### CODE REVIEW

File: src/Views/Results/Filters/Location.cshtml [hunk 1]
```diff
@@ -6,7 +6,9 @@
   </h2>
   <p class="govuk-body filter-form__value">
     @if(Model.Coordinates != null && Model.RadiusOption != null) {
-      <text>@Model.rad miles around @Model.loc</text>
+      @Model.loc
+      <span class="govuk-caption-m">within @Model.rad miles</span>
```
Comment (fofr) [inline src/Views/Results/Filters/Location.cshtml]: Can this say:
"Within @Model.rad miles of the pin"

**Comment by fofr** [comment]
On the prototype we moved the location map outside of the filters disclosure on mobile to make the searched for location and map more prevalent. Should we still do this? @tvararu @dankmitchell

Maybe not until the map is interactive? 

screen shot 2018-11-21 at 12 54 58

File: src/Views/Results/Filters/Location.cshtml [hunk 2]
```diff
@@ -6,7 +6,9 @@
...
+      <img src="https://maps.googleapis.com/maps/api/staticmap?key=@(Environment.GetEnvironmentVariable("google_cloud_platform_key_maps"))&center=@(Model.Coordinates.Latitude),@(Model.Coordinates.Longitude)&zoom=@(Model.GetGoogleMapRadius())&size=300x200&markers=@(Model.Coordinates.Latitude),@(Model.Coordinates.Longitude)" class="google-map-static"/>
```
Comment (fofr) [inline src/Views/Results/Filters/Location.cshtml]: Image needs an alt attribute.

**Comment by tvararu** [comment]
@fofr I think it's fine for now, I agree we can do it when it's a clickable link.

**Pull request merged**